apps/sms/views/new_message/style/recipients.css

```diff
@@ -97,7 +97,7 @@ Do not remove.
   width: auto;
 }
 
-.recipient[contenteditable].email {
+.recipient[contenteditable=false].email {
```

Comment:
It's in BB, but at the same time I don't see that anyone except us using ```.bb-editable``` selector in gaia. So maybe it's safe enough to do this change.